site2/docs/admin-api-overview.md

```diff
@@ -1,39 +1,34 @@
 ---
 id: admin-api-overview
-title: The Pulsar admin interface
+title: Pulsar admin interface
 sidebar_label: Overview
 ---
 
-The Pulsar admin interface enables you to manage all of the important entities in a Pulsar [instance](reference-terminology.md#instance), such as [tenants](reference-terminology.md#tenant), [topics](reference-terminology.md#topic), and [namespaces](reference-terminology.md#namespace).
+The Pulsar admin interface enables you to manage all important entities in a Pulsar instance, such as tenants, topics, and namespaces.
 
-You can currently interact with the admin interface via:
+You can interact with the admin interface via:
 
-- Making HTTP calls against the admin {@inject: rest:REST:/} API provided by Pulsar [brokers](reference-terminology.md#broker). For some restful apis, they might be redirected to topic owner brokers for serving
-   with [`307 Temporary Redirect`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), hence the HTTP callers should handle `307 Temporary Redirect`. If you are using `curl`, you should specify `-L`
-   to handle redirections.
-- The `pulsar-admin` CLI tool, which is available in the `bin` folder of your [Pulsar installation](getting-started-standalone.md):
-
-```shell
-$ bin/pulsar-admin
-```
+- Making HTTP calls against the admin {@inject: rest:REST:/} API provided by Pulsar brokers. For some restful APIs, they might be redirected to the topic owner brokers for serving with [`307 Temporary Redirect`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), hence the HTTP callers should handle `307 Temporary Redirect`. If you use `curl`, you should specify `-L` to handle redirections.
+- A Java client interface.
+- The `pulsar-admin` CLI tool, which is available in the `bin` folder of your Pulsar installation:
 
-Full documentation for this tool can be found in the [Pulsar command-line tools](reference-pulsar-admin.md) doc.
+    ```shell
+     $ bin/pulsar-admin
+    ```
 
-- A Java client interface.
+    For details of `pulsar-admin` tool, see the [Pulsar command-line tools](reference-pulsar-admin.md) doc.
 
-> #### The REST API is the admin interface
-> Under the hood, both the `pulsar-admin` CLI tool and the Java client both use the REST API. If you’d like to implement your own admin interface client, you should use the REST API as well. Full documentation can be found here.
 
-In this document, examples from each of the three available interfaces will be shown.
+> **The REST API is the admin interface**. Both the `pulsar-admin` CLI tool and the Java client use the REST API. If you implement your own admin interface client, you should use the REST API. 
```

Comment:
REST or RESTful ?

Reply:
Here we can use "REST".
RESTful service is the one that implements the REST architectural pattern, and on our REST API page, we use the word "REST".